internal/server/metrics.go

```diff
@@ -0,0 +1,40 @@
+package server
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/livepeer/stream-tester/internal/metrics"
+)
+
+type MetricsServer struct {
```

Comment:
```suggestion
// NewMetricsServer object
type MetricsServer struct {
```

Reply:
hmm, I find this comment redundant. Is this a convention we're following?

Reply:
`golint internal/server/metrics.go`